pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -100,90 +100,96 @@ func (r *ReconcileMasterUserRecord) Reconcile(request reconcile.Request) (reconc
 			return reconcile.Result{}, nil
 		}
 		// Error reading the object - requeue the request.
-		reqLogger.Error(err, "unable to get MasterUserRecord")
+		logger.Error(err, "unable to get MasterUserRecord")
 		return reconcile.Result{}, err
 	}
+	logger.Info("found MasterUserRecord", "mur", mur)
```

Comment:
Do we really need to log the entire mur here and in the other places? I think printing the mur name should be enough. WDYT? And the mur name is already added to the logger in https://github.com/codeready-toolchain/host-operator/pull/165/files#diff-a7167111790d962cde50e8d4ded9aba5R89 as request.Name. So, we should be good just with:
```suggestion
	logger.Info("found MasterUserRecord")
```
So, we can avoid polluting our logs by printing big resources.

Reply:
right, this was not intended here. This log statement is helping me trying to debug another issue, but it's only temporary. Let me remove it at all from this commit.

Reply:
remove in 688a92d